build/container/version

```diff
@@ -1 +1 @@
-0.3.0
\ No newline at end of file
+0.3.1
```

Comment:
ww will need to push the new container to quay.io prior to jenkins build this. I think you need privileges to do so. send me your quay.io user id.